macosx/VDKQueue/VDKQueue.mm

```diff
@@ -169,22 +169,23 @@ - (VDKQueuePathEntry*)addPathToQueue:(NSString*)path notifyingAbout:(u_int)flags
 
 - (void)watcherThread:(id)sender
 {
-    int n;
-    struct kevent ev;
+#if DEBUG_LOG_THREAD_LIFETIME
+    NSLog(@"watcherThread started.");
+#endif
+
+    NSThread.currentThread.name = @"VDKQueue";
+
     // 1 second timeout. Should be longer, but we need this thread to exit when a kqueue is dealloced, so 1 second timeout is quite a while to wait.
-    struct timespec timeout = { 1, 0 };
+    const struct timespec timeout = { 1, 0 };
     // So we don't have to risk accessing iVars when the thread is terminated.
-    int theFD = _coreQueueFD;
+    int const theFD = _coreQueueFD;
 
     NSMutableArray* notesToPost = [[NSMutableArray alloc] initWithCapacity:5];
 
-#if DEBUG_LOG_THREAD_LIFETIME
-    NSLog(@"watcherThread started.");
-#endif
-
     while (_keepWatcherThreadRunning)
     {
-        n = kevent(theFD, NULL, 0, &ev, 1, &timeout);
+        struct kevent ev;
+        int n = kevent(theFD, NULL, 0, &ev, 1, &timeout);
```

Comment:
Isn't it allocating `kevent` on each iteration now?

Reply:
Don't know. I moved it back outside the block to be sure.